Contribute/content/provide-feedback.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Provide feedback for Microsoft Learn content
+description: Learn how to provide feedback to Microsoft Learn content teams using the new feedback experience for documentation and training content.
+author: carlyrevier
+ms.author: cahublou
+ms.topic: contributor-guide
+ms.prod: non-product-specific
+ms.custom: external-contributor-guide
+ms.date: 02/22/2024
+---
+
+# Provide feedback for Microsoft Learn content
+
+This article explains how to provide feedback for Microsoft Learn content using the new feedback experience. This unified feedback experience is now standard across all products that publish documentation or training content on Microsoft Learn. The improved system:
```

Comment:
in a way this has nothing to do with products.  Can we just say "..standarad across all documentation or training content on Microsoft Learn".  Also, lets say "New" system not improved.  we have already been told by several customers this is far from an improvement.